Jenkinsfile

```diff
@@ -27,6 +24,9 @@ pipeline {
         stage('Deploy') {
             steps {
                 sh 'gradle --init-script init-ci.gradle publish'
+                sh 'git clone git://github.com/ome/build-infra .build'
```

Comment:
The default job pipeline config includes `git clean` after checkout or the main repo, but it won't remove nested git clones like this, which is why I'm using the tar archive in https://github.com/ome/omero-build/blob/34f8faf49f7c7aeda1a1b454e791c8adba4e59bb/Jenkinsfile.merge#L35

Reply:
Having said that this job runs `deleteDir()` so it doesn't matter